index.js

```diff
@@ -1,51 +1,55 @@
 'use strict';
 
 var through = require('through2');
-var removeBom = require('remove-bom-buffer');
+var TextDecoder = require('util').TextDecoder;
 
-function removeBomStream() {
-  var state = 0; // 0:Not removed, -1:In removing, 1:Already removed
-  var buffer = Buffer.alloc(0);
+var BOM = Buffer.from([0xEF, 0xBB, 0xBF], 'utf-8');
```

Comment:
```suggestion
var BOM = '\ufeff'
```
Can we compare the string?